src/services/billing-services/list-payment-service.js

```diff
@@ -0,0 +1,67 @@
+import { AxiosHttpClientAdapter, parseHttpResponse } from '../axios/AxiosHttpClientAdapter'
+import { makePaymentBaseUrl } from './make-payment-base-url'
+
+export const listPaymentService = async () => {
+  let httpResponse = await AxiosHttpClientAdapter.request({
+    url: `${makePaymentBaseUrl()}/credit_cards?page_size=200`,
+    method: 'GET'
+  })
+
+  httpResponse = adapt(httpResponse)
+
+  return parseHttpResponse(httpResponse)
+}
+
+const formatDate = (month, year) => {
+  const formatMonth = month < 10 ? `0${month}` : month
+
+  return `${formatMonth}/${year}`
+}
+
+const isExpired = (month, year) => {
+  const currentDate = new Date()
+  const currentYear = currentDate.getFullYear()
+  const currentMonth = currentDate.getMonth() + 1
+
+  if (year < currentYear || (year === currentYear && month < currentMonth)) {
```

Comment:
Shouldn't we flip this?
```suggestion
  if (year > currentYear || (year === currentYear && month > currentMonth)) {
```

Reply:
To improve the readability and clarity of your code, you can break down the condition into intermediate variables.
```suggestion
  const isYearInThePast = year < currentYear;
  const isMonthInThePast = year === currentYear && month < currentMonth;

  return isYearInThePast || isMonthInThePast ? 'Expired' : ''
```

Reply:
funciona corretamente

Reply:
Essa sugestão não valida corretamente a expiração do cartão